paparazzi/build.gradle

```
@@ -72,8 +72,12 @@ subprojects {
       }
     }
 
```

You probably want something like this here to still sign artifacts and add sonatype/central as a publishing repo
```suggestion
    mavenPublishing {
        publishToMavenCentral()
        signAllPublications()
    }
```
Alternatively you can also set to Gradle properties like described here https://github.com/vanniktech/gradle-maven-publish-plugin/blob/master/CHANGELOG.md#version-0210-2022-07-11

---

oh, i had done that (the 2nd approach), but looks like i forgot to add it.  that's what I get for late night PRs.  thanks for the sanity check!